03_flow_MEFL/exercises.m

```diff
@@ -94,7 +107,7 @@
 % When peaks are mis-detected, this typically leads to a bad fit against the
 % expected sequence gaps, giving warning of failures.
 
-TASBEConfig.set('beads.rangeMin', 2); % Don't consider beads less than this (log10) amount
+TASBEConfig.set('beads.rangeMin', 2); % Don't consider beads less than this 10^2 amount
 TASBEConfig.set('beads.peakThreshold', 200); % override default peak threshold
```

Comment:
Do you want to put these plots in a different folder than the "broken" color model plots, so that both are directly available to a student?

Reply:
done